Jenkinsfile-deploy

```diff
@@ -28,21 +28,20 @@ pipeline {
             steps {
                 script {
                     unstash "consignment-jar"
-                    def accountNumber = getAccountNumberFromStage()
+                    def accountNumber = tdr.getAccountNumberFromStage(params.STAGE)
                     sh "python3 /deploy_lambda.py ${accountNumber} ${params.STAGE} tdr-database-migrations-${params.STAGE} lambda/target/scala-2.13/consignment.jar"
                     slackSend color: "good", message: "*Database migrations* :arrow_up: The migration lambda has been deployed to *${params.STAGE}*", channel: "#tdr-releases"
                 }
             }
         }
     }
-}
-
-def getAccountNumberFromStage() {
-    def stageToAccountMap = [
-            "intg": env.INTG_ACCOUNT,
-            "staging": env.STAGING_ACCOUNT,
-            "prod": env.PROD_ACCOUNT
-    ]
-
-    return stageToAccountMap.get(params.STAGE)
+    post {
+        success {
+            script {
+                if (params.STAGE == "intg"){
+                    tdr.runEndToEndTests(300, params.STAGE, BUILD_URL)
```

Comment:
Same question as for the Terraform jobs: maybe we don't need a delay in this case because the changes take effect immediately?

Reply:
Think you're right for this one. There wouldn't be a delay